Jenkinsfile

```
@@ -12,7 +12,7 @@ def rubyDockerBuildAndPush(rubyVersion, branch) {
 def dockerBuildAndPush(image, branch, buildArgs = '', versionPrefix = '') {
   def version = sh(returnStdout: true, script: "cat ${image}/VERSION").trim()
   def tag = "docker.voxops.net/${image}:${versionPrefix}${version}"
-  sh "docker build ${image} ${buildArgs} -t ${tag}"
+  sh "docker build ${image} ${buildArgs} -t ${tag} --build-arg IMAGE_VERSION=${version}"
```

So it looks like this is adding this arg to _every_ base image, not just the ruby base image. Is it silently ignored for other base images?

---

Both the capistrano and the local-dns images have a `VERSION` file that will be pulled in. It will assign that value to the build argument - so that part is ok.

Let's see what happens when I try and running that command locally without an `ARG` in the docker image:
```
🤔 👉 docker build ./ruby --build-arg MOO=COW -t base-image-test
```
Ignores the `MOO=COW` build arg.

I think this is safe.

---

It raises a warning, but completes successfully.
```
 ---> 3a56b38f4140
[Warning] One or more build-args [MOO] were not consumed
Successfully built 3a56b38f4140
Successfully tagged base-image-test:latest
```